locales/en-US/server.properties

```diff
@@ -213,6 +213,7 @@ snippetJSChangeStyleComment=Select the element with id='alert'
 
 # Publishing
 publishHeader=Publish your Project
+publisedhHeader=Your Project is Online
```

Comment:
since you're using this in the client files you should add this string to the client.properties file instead as per https://github.com/mozilla/thimble.mozilla.org/wiki/Localization#client-side-localization.
Also rename this to `publishHeaderOnline`